src/Controls/src/Xaml/MarkupExtensions/OnIdiomExtension.cs

```diff
@@ -121,4 +121,115 @@ object GetValue()
 			return Default;
 		}
 	}
+
+	// NOTE: currently internal, to be used by XamlC compiler
+	[ContentProperty(nameof(Default))]
+	[RequireService(
+		[typeof(IProvideValueTarget),
+		 typeof(IValueConverterProvider),
+		 typeof(IXmlLineInfoProvider),
+		 typeof(IConverterOptions)])]
+	internal class OnIdiomExtension<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] T> : IMarkupExtension
```

Comment:
if it's used by generated code, it has to be public (the tests passes cause there's an internalsvisibleto for the unit tests project)

you can decorate it with a [EditorBrowseable] attribute, or even [Obsolete] it to prevent usage... or move it to a namespace with 'Internals' in the name...

Reply:
So, this might be the cause of the current CI failures. I'll try a `public` type in the `*.Internals` namespace.